go/libraries/doltcore/dbfactory/gs.go

```diff
@@ -17,9 +17,11 @@ package dbfactory
 import (
```

Comment:
This file should be gcs.go, right?

Reply:
And the struct / methods should be GCSFactory, right?

Reply:
Google uses the scheme gs:// for their GCS urls.  These factories refer to the schemes.  It is a factory for creating databases backed by the GS scheme.